README.md

```diff
@@ -119,7 +119,7 @@ npm install
 cd ../
 git clone https://github.com/kubernetes-client/gen
```

Comment:
Totally separate from this PR, but @silasbw @brendanburns should we not be version locking this so it's reproducible?

Maybe add a CI check to verify that the generated files match the version in "code"

Reply:
I am going to resolve this in favour of an issue #325 325

Reply:
This makes sense to me.